reana_workflow_controller/rest.py

```diff
@@ -1526,3 +1644,13 @@ def _get_current_job_progress(workflow_id):
                 'prettified_cmd': job.prettified_cmd,
                 'current_job_name': job.name}
     return current_job_commands
+
+
+def _get_workflow_input_parameters(workflow):
```

Comment:
Shall we move this logic to the `Workflow` model on a new method?

Reply:
IMO this function is too specific to be in models and could cause confusion. Currently it is used only by serial. Once we have option to override input parameters for the other engines, we could move it to models. What do you think?

Reply:
Agreed! All ready to test run?